docs/examples/add-to-your-application/ruby/routes.rb

```diff
@@ -0,0 +1,27 @@
+require 'oso'
+require 'sinatra'
+
+require_relative './oso'
+require_relative './models'
+
+class User
```

Comment:
Maybe this doesn't need to be a class method? just `def get_current_user` at the base level of this file?

Reply:
I was aiming to match the route from the quickstart... but maybe not necessary